cloudify_suspend/README.md

```diff
@@ -1,15 +1,82 @@
 # Cloudify Utilities: Suspend
-Additional suport for `suspend`/`resume` workflows.
+Additional suport for `suspend`, `resume`, `backup`, `restore`, `remove_backup`
+workflows.
 
 ## Supported workflows
 
-* `suspend`: Workflow call `cloudify.interfaces.lifecycle.suspend` for each node that have such operation.
-* `resume`: Workflow call `cloudify.interfaces.lifecycle.resume` for each node that have such operation.
+* `suspend`: Workflow call `cloudify.interfaces.freeze.suspend` for each node
+  that has such operation. For backward compatibility also run deprecated
+  `cloudify.interfaces.lifecycle.suspend`.
+* `resume`: Workflow call `cloudify.interfaces.freeze.resume` for each node
+  that has such operation. For backward compatibility also run deprecated
+  `cloudify.interfaces.lifecycle.resume`.
+* `backup`: Workflow call such calls for each node that has such operation.
+  * `cloudify.interfaces.freeze.fs_finalize` for all non compute nodes,
+  * `cloudify.interfaces.freeze.fs_finalize` for all compute nodes,
+  * `cloudify.interfaces.snapshot.create` for all nodes in deployment,
+  * `cloudify.interfaces.freeze.fs_finalize` for all compute nodes,
+  * `cloudify.interfaces.freeze.fs_finalize` for all non compute nodes.
```

Comment:
``
`cloudify.interfaces.freeze.fs_finalize` for all compute nodes,
`cloudify.interfaces.freeze.fs_finalize` for all non compute nodes.
``

I noticed the above are duplicated